spec/pluto/image_spec.cr

```diff
@@ -1,6 +1,6 @@
 require "../spec_helper"
 
-describe Pluto::RGBAImage do
+Spectator.describe Pluto::RGBAImage do
```

Comment:
Other than this change, all other specs were untouched and continue to function as before with no changes.